### PR TITLE
add DC Onchain to community meetups

### PR DESCRIPTION
**Pull request by dswilson4**
Add DC Onchain to the meetups json data

### CODE REVIEW

File: src/data/community-meetups.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+    {
+    "title": "DC Onchain",
+    "emoji": ":us:",
+    "location": "DC",
```
Comment (minimalsm) [inline src/data/community-meetups.json:5]: @wackerow thoughts on location 🤔?

```suggestion
    "location": "Arlington, Virginia",
```

**Comment by dswilson4** [comment]
@minimalsm committed your suggestion, possible the location might move around VA/DC/MD but that is TBD and I could make a PR later in the future if that actually changes

**Comment by dswilson4** [comment]
@minimalsm for visibility

**Review by minimalsm — APPROVED** [review]
Sorry for delay, approving and merging.

Thanks for the PR @dswilson4!

**Comment by minimalsm** [comment]
@all-contributors please add @dswilson4 for content

**Pull request merged**